user/services/UserManager.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import User #this will be replaced by our class user
```

Comment:
consider renaming to e.g. user_service.py because managers are reserved to model managers in models subpackage